tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/provision_esxi.yml

```diff
@@ -4,15 +4,14 @@
     name: cloud.vmware_ops.provision_virtual_esxi
 
 - name: Wait until ESXI VM is powered off after OS installation
-  community.vmware.vmware_guest_info:
+  vmware.vmware.guest_info:
     hostname: "{{ provision_virtual_esxi_hostname }}"
     username: "{{ provision_virtual_esxi_username }}"
     password: "{{ provision_virtual_esxi_password }}"
-    datacenter: "{{ provision_virtual_esxi_datacenter }}"
     validate_certs: "{{ provision_virtual_esxi_validate_certs }}"
     name: "{{ provision_virtual_esxi_vms[0].name }}"
   register: _esxi_power_status
-  until: _esxi_power_status.instance.hw_power_status == 'poweredOff'
+  until: _esxi_power_status.guests[0].hw_power_status == 'poweredOff'
```

Comment:
Did we break that?

Reply:
No, the return value for `guest_info` hasnt changed.

`instance` is the return value used in `community.vmware.guest_info`. 
`guests` is the return value that has been used in the original and now new version of `vmware.vmware.guest_info`